src/mcts/search.cc

```diff
@@ -682,7 +682,25 @@ void SearchWorker::ExtendNode(Node* node) {
       return;
     }
   }
-
+  
+  //WDL probing
+  if(board.our + board.their <= SyzygyTablebase::max_cardinality && board.GetNoCapturePly == 0
+     && board.no_legal_castle()) {
+    SyzygyTablebase::ProbeState* result;
```

Comment:
ProbeState is currently just in the lczero namespace in my PR - so this won't need namespace qualification.